### ISSUE TITLE
estimating uncertainty using MCMC

### ISSUE THREAD

**Issue by davidjpmoore** [davidjpmoore/Calloc_AmerifluxProcessing#2]
@fmontane5 
I had a chat with @malloryb today and we worked through some issues with the gap filling techniques. She will be forwarding you some issues - regarding sites that she was unable to partition using the code. 

I wanted to bring up one more issue. (not urgent right now)
I think the only way to estimate the UNCERTAINTY of GPP using the techniques we've been trying out is to calculate GPP from a distribution of ustar values. I originally thought we would be able to use just the percentiles but I think we'll need to rethink that.

**Comment by fmontane5** [comment]
@davidjpmoore @malloryb 

Ok. We can chat about these issues.

I competed partition for Niwot between 1999 and 2013 using Ustar 5% bootstrap, Ustar 50% bootstrap and Ustar 95% bootstrap. Even that the Ustar 50% is always between Ustar5% and Ustar95%, what happens is that for some years GPP 50% is not between GPP 5% and GPP 95%. For instance year 1999, GPP_U05=1154.8, GPP_U50=1165.9 and GPP_U95=1115.3. The same thing happens some years for NEE and Reco. Perhaps using a distribution of Ustar values may solve this.

**Comment by davidjpmoore** [comment]
Yes that’s what I thought too – below is the difference between 95% and 5% GPP estimates … in 2 different years the 5%>95%
I think this means we need a distribution to estimate confidence intervals.

Dave

1999

-39.5

2000

55.45

2001

54.79

2002

79

2003

43.89

2004

388.81

2005

-37.21

2006

36.89

2007

53.51

2008

40.89

2009

61.27

2010

42.27

2011

85.73

2012

32.38

2013

73.96

**Comment by fmontane5** [comment]
It seems that Papale et al. 2006, used the Ustar thresholds at 5%, 50% and 95% to estimate uncertainty on annual GPP (Figure 9c).

**Comment by davidjpmoore** [comment]
I’ll ask Dario when I see him.
